# Request
---
# Response
int16[6] curpos
